tests/mock_crust/churn.rs

```diff
@@ -90,6 +92,44 @@ fn did_receive_get_success(node: &TestNode,
     }
 }
 
+fn quorum_before(nodes: &[TestNode],
+                 src: Authority,
+                 dst: Authority,
+                 data_id: DataIdentifier,
+                 message_id: MessageId)
+                 -> usize {
+    let group = nodes.iter()
+        .filter(|n| n.routing_table().is_recipient(&src.to_destination()))
+        .collect_vec();
+
+    for node in &group {
+        let _ = node.inner.send_get_request(src, dst, data_id, message_id);
+    }
+
+    (group.len() * QUORUM - 1) / 100 + 1
+}
```

Comment:
Since it's just three lines, I'd go with the duplication. If not, I'd still put these into a separate function - then it's just one more line that's duplicated.
In the "node to group" scenario we even _need_ these to be separate, I think: We're sending from a node, so we don't need 105-107, but we're receiving at a group, so we _do_ need to compute the before and after quora.